content/zh-cn/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -849,7 +849,7 @@ kubectl delete pod security-context-demo-4
 * [使用最新的安全性增强来调优 Docker（英文）](https://github.com/containerd/containerd/blob/main/docs/cri/config.md)
 * [安全上下文的设计文档（英文）](https://github.com/kubernetes/design-proposals-archive/blob/main/auth/security_context.md)
 * [属主管理的设计文档（英文）](https://github.com/kubernetes/design-proposals-archive/blob/main/storage/volume-ownership-management.md)
-* [Pod 安全性准入](zh-cn/docs/concepts/security/pod-security-admission/)
+* [Pod 安全性准入](https://kubernetes.io/zh-cn/docs/concepts/security/pod-security-admission/)
```

Comment:
```suggestion
* [Pod 安全性准入](/zh-cn/docs/concepts/security/pod-security-admission/)
```
Original is correct.

Reply:
Right, a leading `\` is missing. That is all.

Reply:
Changed.

Reply:
It's still not correct. The green line is my suggestion. You can directly copy/paste the suggested change if you accept it.